build.gradle

```diff
@@ -4,37 +4,14 @@ equoIde {
 	welcome().openUrl('https://github.com/diffplug/spotless/blob/main/CONTRIBUTING.md')
 	gradleBuildship().autoImport('.')
 }
-
 repositories {
 	mavenCentral()
 }
-
-apply from: rootProject.file('gradle/java-publish.gradle')
-apply from: rootProject.file('gradle/changelog.gradle')
 allprojects {
 	apply from: rootProject.file('gradle/error-prone.gradle')
-	apply from: rootProject.file('gradle/rewrite.gradle')
```

Comment:
need this only once in root. Then the dependencies can be moved into dedicated file like done in other projects i was always wondering why we can not move it here.